### DISCUSSION TITLE
Where Do I put Download Proxy Url ??

### DISCUSSION THREAD

**Discussion by anymeofu** [AlistGo/alist#4021] [Q&A] [answered]
I am trying to use **cloudflare workers** proxy to get optimum download speeds!

I was following the documentation https://alist.nn.ci/guide/drivers/common.html#download-proxy-url
I am unable to understand the 3rd point 

> When mounting the configuration in the Alist background, fill in the download proxy URL at the end of the link without /

I have setup alist in a docker on my VPS and my config looks like this:

```
{
  "force": false,
  "address": "0.0.0.0",
  "port": 5244,
  "site_url": "https://alist.website.com",
  "cdn": "",
  "jwt_secret": "z30*********",
  "token_expires_in": 48,
  "database": {
    "type": "sqlite3",
    "host": "",
    "port": 0,
    "user": "",
    "password": "",
    "name": "",
    "db_file": "data/data.db",
    "table_prefix": "x_",
    "ssl_mode": ""
  },
  "scheme": {
    "https": false,
    "cert_file": "",
    "key_file": ""
  },
  "temp_dir": "data/temp",
  "bleve_dir": "data/bleve",
  "log": {
    "enable": true,
    "name": "data/log/log.log",
    "max_size": 10,
    "max_backups": 5,
    "max_age": 28,
    "compress": false
  },
  "max_connections": 0,
  "tls_insecure_skip_verify": true
}
```

**Comment by Rektyfikowany** [comment, marked as answer]
You can fill it in when you add a new storage.
![prtsc0402](https://user-images.githubusercontent.com/126593039/229330439-148d0bb1-17b3-4c69-938c-0a3275ad51b5.png)
It will be stored in `data.db` rather than `config.json`.

**Reply by anymeofu** [reply]
Thanks Got it !
